migtests/tests/oracle-tests/ora-datatypes-test/env.sh

```diff
@@ -0,0 +1,10 @@
+export SOURCE_DB_TYPE="oracle"
+export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"ORCL"}
+export SOURCE_DB_HOST=${SOURCE_DB_HOST:-"10.9.8.213"}
+export SOURCE_DB_PORT=${SOURCE_DB_PORT:-1521}
+export SOURCE_DB_USER=${SOURCE_DB_USER:-"ADMIN"}
+export SOURCE_DB_PASSWORD=${SOURCE_DB_PASSWORD:-"Password1"}
+export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"ADMIN"}
+export TARGET_DB_HOST=${TARGET_DB_HOST:-"10.9.203.12"}
+export TARGET_DB_NAME=${TARGET_DB_NAME:-"datatypes_test"}
+export TARGET_DB_PASSWORD=${TARGET_DB_PASSWORD:-"password"}
```

Comment:
In this test specific `env.sh`, you need to pass only those variable which have to be changed for this particular test and others are not required it will take from the parent `env.sh`.